Sources/XCParseCore/XCResultToolCommand.swift

```diff
@@ -100,6 +103,14 @@ open class XCResultToolCommand {
             let process = TSCBasic.Process(arguments: processArgs)
             super.init(withXCResult: xcresult, process: process)
         }
+
+        public override func run() -> ProcessResult? {
+            let superResult = super.run()
+            if superResult == nil { return nil }
+            guard let timestamp else { return superResult }
```

Comment:
```
/Users/vagrant/git/Sources/XCParseCore/XCResultToolCommand.swift:110:33: error: variable binding in a condition requires an initializer
            guard let timestamp else { return superResult }
                                ^
```

Our build system is flagging this as this seems to be [new syntax that requires Swift 5.7](https://www.swiftbysundell.com/articles/swifts-new-shorthand-optional-unwrapping-syntax/). To ensure backwards compatibility of xcparse with older Xcode versions, could you modify this to ensure compilation on Swift 5?

Think it should be something like:

```suggestion
            guard let timestamp = timestamp else { return superResult }
```

Reply:
Ahh I was wondering why the build failed :-) I'll fix!